guides/common/modules/proc_changing-the-mirroring-policy-for-a-repository.adoc

```diff
@@ -0,0 +1,11 @@
+[id="Changing_the_Mirroring_Policy_for_a_Repository_{context}"]
+= Changing the Mirroring Policy for a Repository
+
+
+You can set the mirroring policy for a repository.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Products*.
+. Select the product name.
+. On the *Repositories* tab, click the repository name, locate the *Mirroring Policy* field, and click the edit icon.
+. From the list, select the required mirroring policy and then click `Save`.
```

Comment:
```suggestion
. From the list, select a mirroring policy and click *Save*.
```

Elements from the Foreman Web UI are always bold (`*`), user input is generally in fixed width fonts.